tests/test_utils/test_schemaview.py

```diff
@@ -165,525 +166,277 @@ def sv2() -> SchemaView:
+    view = schema_view_with_imports
+    assert view.schema.source_file is not None
     logger.debug(view.imports_closure())
-    assert len(view.imports_closure()) == 1
+    assert set(view.imports_closure()) == {"kitchen_sink", "core", "linkml:types"}
 
-    all_cls = view.all_classes()
-    logger.debug(f"n_cls = {len(all_cls)}")
+    for t in view.all_types().keys():
+        logger.debug(f"T={t} in={view.in_schema(t)}")
+    assert view.in_schema(ClassDefinitionName("Person")) == "kitchen_sink"
+    assert view.in_schema(SlotDefinitionName("id")) == "core"
+    assert view.in_schema(SlotDefinitionName("name")) == "core"
+    assert view.in_schema(SlotDefinitionName(ACTIVITY)) == "core"
+    assert view.in_schema(SlotDefinitionName("string")) == "types"
 
-    assert list(view.annotation_dict(IS_CURRENT).values()) == ["bar"]
-    logger.debug(view.annotation_dict(EMPLOYED_AT))
-    e = view.get_element(EMPLOYED_AT)
-    logger.debug(e.annotations)
-    e = view.get_element("has employment history")
-    logger.debug(e.annotations)
+    assert ACTIVITY in view.all_classes()
+    assert ACTIVITY not in view.all_classes(imports=False)
+    assert "string" in view.all_types()
+    assert "string" not in view.all_types(imports=False)
+    assert len(view.type_ancestors("SymbolString")) == len(["SymbolString", "string"])
 
-    elements = view.get_elements_applicable_by_identifier("ORCID:1234")
-    assert "Person" in elements
-    elements = view.get_elements_applicable_by_identifier("PMID:1234")
-    assert "Organization" in elements
-    elements = view.get_elements_applicable_by_identifier("http://www.ncbi.nlm.nih.gov/pubmed/1234")
-    assert "Organization" in elements
-    elements = view.get_elements_applicable_by_identifier("TEST:1234")
-    assert "anatomical entity" not in elements
+    for tn, t in view.all_types().items():
+        assert tn == t.name
+        induced_t = view.induced_type(tn)
+        assert induced_t.uri is not None
+        assert induced_t.base is not None
+        if t in view.all_types(imports=False).values():
+            assert t.from_schema == "https://w3id.org/linkml/tests/kitchen_sink"
+        else:
+            assert t.from_schema in ["https://w3id.org/linkml/tests/core", "https://w3id.org/linkml/types"]
 
-    assert list(view.annotation_dict(SlotDefinitionName(IS_CURRENT)).values()) == ["bar"]
-    logger.debug(view.annotation_dict(SlotDefinitionName(EMPLOYED_AT)))
-    element = view.get_element(SlotDefinitionName(EMPLOYED_AT))
-    logger.debug(element.annotations)
-    element = view.get_element(SlotDefinitionName("has employment history"))
-    logger.debug(element.annotations)
+    for en, e in view.all_enums().items():
+        assert en == e.name
+        if e in view.all_enums(imports=False).values():
+            assert e.from_schema == "https://w3id.org/linkml/tests/kitchen_sink"
+        else:
+            assert e.from_schema == "https://w3id.org/linkml/tests/core"
```

Comment:
amazing diff work here

This test function is completely different to the one git diff thinks it's comparing to (although both are far too long and need to be split out into separate tests).